code/src/components/UserLocation.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+/* eslint-disable react/react-in-jsx-scope */
+const UserLocation = ({ userlocation, setUserlocation }) => {
+  const handleUserLocationInputChange = (event) => {
+    setUserlocation(event.target.value);
+  };
+
+  return (
+    <>
+      <p className="star-shop" tabIndex="0">What is your nearest physical <span className="star">Star shop</span>⭐? </p>
+      <p className="user-p" tabIndex="0">Select one of the options below</p>
```

Comment:
Suggestion would be to use h2-h6 instead of a p-tag here, that way you do not need to add the tabindex. Here's a good article on tabindexes, https://webaim.org/techniques/keyboard/tabindex.